imap_processing/tests/conftest.py

```diff
@@ -443,9 +455,9 @@ def make_data(start_met: float, end_met: Optional[float] = None) -> pd.DataFrame
             end_met = start_met + 86400
 
         # Create spin start second data of 15 seconds increment
-        spin_start_met = np.arange(start_met, end_met + 1, 15)
+        spin_start_met = np.arange(start_met, end_met + 1, spin_period)
```

Comment:
If we are now doing decimal values you probably want to make this a smaller value to add to end_met so you don't accidentally include `100.2` when you had the end at 100.

```suggestion
        spin_start_met = np.arange(start_met, end_met + 0.001, spin_period)
```